docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,107 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+For this example, we will be using the `192.168.1.100` and `fe80::192:168:1:100` addresses for the following steps. You will need to substitute your own addresses in the following steps.
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.100` and `xxxx::192:168:1:100` respectively, this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.
+
+Worth to mention that the `IPv6` address that starts with `fe80` is the `link-local` address and it is used in this guide, the other addresses that start with `2806` are the global addresses and they are assigned by the router.
+
+!!! warning
+    Do not use `127.0.0.1#5335` or `::1#5335` as `DNS server` in the `Raspberry Pi` because it will cause a `DNS loop` as the `Pi-hole` will send the queries to the router which in turn will send them to `Pi-hole`.
```

Comment:
What are you talking about here? Do you have another DNS server running at `127.0.0.1#5335`? Why should this cause a DNS loop?

Reply:
This part is referring to an old configuration in my Raspi, where I was using unbound. I forgot to uninstall unbound and its configurations and hence I was understanding that somehow the Raspi was looping the request due to the `127.0.0.1#5335`. I started again the installation and this time the local IP#5335 is not an issue.

Reply:
Removed the phrase. 
Also I renamed `wlan0` to `eth0` to make the guide homogeneous with current router guides.